game/players.cc

```diff
@@ -30,7 +29,7 @@ void Players::load(xmlpp::NodeSet const& n) {
 		xmlpp::Attribute* a_id = element.get_attribute("id");
 		if (!a_id) throw PlayersException("Attribute id not found");
 		int id = -1;
-		try {id = boost::lexical_cast<int>(a_id->get_value());} catch (boost::bad_lexical_cast const&) { }
+		try { id = std::stoi(a_id->get_value()); } catch (std::exception&) { }
```

Comment:
so you're basically catching every exception here?

Reply:
Better throw a http://www.cplusplus.com/reference/stdexcept/invalid_argument/ exception i think